bench/formal/mor1kx_execute_alu.sby

```diff
@@ -12,8 +12,8 @@ test_sim_mul     simulation_mul    simulation_div   barrel_shift
 mode prove
 depth 10
 
-test_serial: depth 64
-
+test_serial: depth 40
+test_3stage_mul: depth 40
```

Comment:
I think task `test_3stage_mul` requires more depth as it verifies the serial shifter.

Reply:
Fine. I will add a barrel shifter for `test_3stage_mul`